src/components/v5/shared/Stepper/partials/StepperButton/consts.ts

```diff
@@ -5,8 +5,12 @@ export const STEP_STAGE = {
   Current: 'current',
   Upcoming: 'upcoming',
   Skipped: 'skipped',
+  Passed: 'passed',
```

Comment:
Those steps are related to motions only and they won't be needed anywhere else. This component should stay reusable, without any connection to any functionality in the app. Can you use className prop to add needed styles?